src/control/server/mgmt_pool.go

```diff
@@ -175,23 +197,28 @@ func (svc *mgmtSvc) calculateCreateStorage(req *mgmtpb.PoolCreateReq) error {
 		}
 	}
 
-	// zero these out as they're not needed anymore
-	req.Totalbytes = 0
-	for tierIdx := range req.Tierratio {
-		req.Tierratio[tierIdx] = 0
-	}
-
 	targetCount := instances[0].GetTargetCount()
 	if targetCount == 0 {
 		return errors.New("zero target count")
 	}
-	minNvmeRequired := engine.NvmeMinBytesPerTarget * uint64(targetCount)
 
-	if req.Tierbytes[1] != 0 && req.Tierbytes[1] < minNvmeRequired {
-		return FaultPoolNvmeTooSmall(req.Tierbytes[1], targetCount)
+	tgts, ranks := uint64(targetCount), uint64(len(req.GetRanks()))
```

Comment:
why make these uint64 rather than just regular int? seems unnecessary

Reply:
The values involved are a mixture of int and uint64. It seems clearest to just do the cast here to keep the helpers as simple and clear as possible.